executor.go

```diff
@@ -223,15 +224,30 @@ func (e *Executor) Wait(ctx context.Context) (*ExecResult, error) {
 	}
 
 	if e.capture {
-		res.Stdout = string(e.stdoutBuf.Bytes())
-		res.Stderr = string(e.stderrBuf.Bytes())
+		res.Stdout = e.stdoutBuf.String()
+		res.Stderr = e.stderrBuf.String()
 	}
 
 	res.Runtime = e.TimeRunning()
 
 	return res, err
 }
 
+func (e *Executor) flush(bufType string) {
+	r := e.stdout
+	w := &e.stdoutBuf
+	if bufType == "stderr" {
+		r = e.stderr
+		w = &e.stderrBuf
+	}
+
+	s := bufio.NewScanner(r)
+	s.Split(bufio.ScanBytes)
```

Comment:
I also don't quite understand what's happening here. Are you trying to flush by forcing line buffering? This won't work for stderr.